x-pack/plugins/apm/server/lib/helpers/get_ml_client.ts

```diff
@@ -5,13 +5,18 @@
  * 2.0.
  */
 
+import {
+  MlAnomalyDetectors,
+  MlMlSystem,
+  MlModules,
+} from '@kbn/ml-plugin/server';
 import { isActivePlatinumLicense } from '../../../common/license_check';
 import { APMRouteHandlerResources } from '../../routes/typings';
 
 export interface MlClient {
-  mlSystem: any;
-  anomalyDetectors: any;
-  modules: any;
+  mlSystem: MlMlSystem;
```

Comment:
yes it is a bit odd. it's because we append `Ml` to all the types we export, but this one already starts with `Ml`
We discussed at the time and agreed that consistency more important than than oddness.
We can revisit this though.